string input_string
---
string status_message

